builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -21,7 +21,7 @@ local function clients_list_formspec(dialogdata)
 		"size[6,9.5]",
 		TOUCH_GUI and "padding[0.01,0.01]" or "",
 		"hypertext[0,0;6,1.5;;<global margin=5 halign=center valign=middle>",
-			fgettext("This is the list of clients connected to\n$1",
+			fgettext("Players connected to\n$1",
```

Comment:
wouldn't this make more sense
```suggestion
			fgettext("Players connected to $1:",
```

Reply:
The newline was originally added because of the long string since it would look weird on smaller screens.
Probably a decision of taste, personally I prefer the current state.

Reply:
can we still add a `:`?

Reply:
Sure.

Reply:
i prefer the current state tbh, the colon shouldn't be necessary because the styling already communicates the separation. honestly i feel it looks a bit off with the colon:

![Screenshot from 2025-02-14 22-42-46](https://github.com/user-attachments/assets/90f9f7a1-c694-4e8b-ba2a-4bea86438f86)

Reply:
Sorry, just saw your comment. Yes, looks somewhat off indeed.

Reply:
Should I revert the last commit and force push?

Reply:
> Ready to be merged now?

as far as i'm concerned, yes